spec/std/crystal/syntax_highlighter/colorize_spec.cr

```diff
@@ -0,0 +1,126 @@
+require "crystal/syntax_highlighter/colorize"
+require "spec"
+
+private def it_highlights(code, expected, *, file = __FILE__, line = __LINE__)
+  it code.inspect, file, line do
```

Comment:
Previously this was `it "highlights #{code.inspect}"`, probably was better?

Reply:
It's used in the `.highlight` example group, so the label would just be duplicated.

Reply:
I don't think it matter much either way, though.